src/test/java/org/jenkinsci/plugins/github_branch_source/ApiRateLimitCheckerTest.java

```diff
@@ -158,9 +158,14 @@ private void setupStubs(List<RateLimit> scenarios) {
             String limit = Integer.toString(scenarioResponse.limit);
             String remaining = Integer.toString(scenarioResponse.remaining);
             String reset = Long.toString(scenarioResponse.reset.toInstant().getEpochSecond());
-            String body = "{ \"rate\": { \"limit\": " + limit + ", " +
-                    "\"remaining\": " + remaining + ", " +
-                    "\"reset\": " + reset + " } }";
+            String body = "{" +
```

Comment:
The new version of github-api uses the `resources.core` member to get rate limit info (the `rate` member is deprecated). This test wasn't creating that member.